knowledge/project-specific/file-organization.md

```
@@ -0,0 +1,389 @@
+├── function_file.c     # Function implementations
+└── helper.h            # Function declarations
+```
+
+### With Testing
+
+Projects with tests include additional directories:
+
+```
+project_name/
+├── main.c              # Entry point
+├── function_file.c     # Implementations
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)